mlrun/projects/pipelines.py

```diff
@@ -319,8 +319,8 @@ def enrich_function_object(
     f.metadata.project = project.metadata.name
     setattr(f, "_enriched", True)
 
-    # set project default image if defined and function does not have an image specified
-    if project.spec.default_image and not f.spec.image:
+    # set project default image if defined and function does not have an image/base image specified
```

Comment:
mind you please change the comment to the reasons behind?
the comment itself simply perform human translation of the `if` which, is not really helpful.
perhaps after digging enough you could add explanation of what is the reasons behind